src/components/nodes/signalEndEvent/index.js

```diff
@@ -0,0 +1,21 @@
+import component from './signalEndEvent.vue';
+import endEventConfig from '../endEvent/index';
```

Comment:
When you point an import to a folder and don't specify a filename, it will assume `index.js` by default; this is what we've been doing in all the other "index" node files. It might be worth switching this to keep it consistent, but at the same time, I kind of like explicitly adding the filename.

Reply:
I'll switch to the standard here :)